regression/contracts/loop_assigns_inference/test.desc

```diff
@@ -0,0 +1,18 @@
+CORE
+main.c
+--apply-loop-contracts
+^EXIT=0$
+^SIGNAL=0$
+^\[body_1.assigns.\d+\] .* Check that j is assignable: SUCCESS$
+^\[body_2.assigns.\d+\] .* Check that \*i is assignable: SUCCESS$
+^\[body_3.assigns.\d+\] .* Check that \*i is assignable: SUCCESS$
+^\[incr.assigns.\d+\] .* Check that \*i is assignable: SUCCESS$
+^\[main.\d+\] .* Check loop invariant before entry: SUCCESS$
+^\[main.\d+\] .* Check that loop invariant is preserved: SUCCESS$
+^\[main.assertion.\d+\] .* assertion j == 9: SUCCESS$
+^VERIFICATION SUCCESSFUL$
+--
+--
+This test checks write set inclusion checks in presence of function calls,
+which are inlined, and also checks that DEAD instructions introduced during
```

Comment:
Can you add some regex in the test description to check the DEAD instructions? We always print out the GOTO program for all tests.

Reply:
This test is taken from the test `loop_assigns-05` by removing assigns clause. Sorry I forgot to modify the text description. 
BTW, I believe that no-failed-check is enough to show that DEAD instruction is correctly positioned. Otherwise the validity checks for locals of inlined functions will fail.